src/containers/HomeV2/HomeV2.js

```diff
@@ -69,40 +76,48 @@ export class HomeV2 extends React.Component {
           .filter(key => temporaryRecommendationsKinds.includes(key))
           .map(recommendationKey => {
             let recommendedData = this.reformatRecommendation(
-              userRecommendations[recommendationKey] , recommendationKey
+              userRecommendations[recommendationKey],
+              recommendationKey
             );
-            if(["unSolvedPySkills", "solvedPySkills"].includes(recommendationKey)){
-              let uniqueProblems = {};
-              recommendedData.forEach(data=>{
-                if(uniqueProblems[data.actualProblem]){
-                  uniqueProblems[data.actualProblem].push(`${data.feature} ${ data.featureType}`);
-                }else{
-                  uniqueProblems[data.actualProblem] = [`${data.feature} ${ data.featureType}`];
-                  uniqueProblems[data.actualProblem].data = {...data};
+            if (
+              ["unSolvedPySkills", "solvedPySkills"].includes(recommendationKey)
+            ) {
+              let uniqueActivities = {};
+              recommendedData.forEach(data => {
+                if (uniqueActivities[data.actualProblem]) {
+                  uniqueActivities[data.actualProblem].push(
+                    `${data.feature} ${data.featureType}`
+                  );
+                } else {
+                  uniqueActivities[data.actualProblem] = [
+                    `${data.feature} ${data.featureType}`
+                  ];
+                  uniqueActivities[data.actualProblem].data = { ...data };
                 }
-              })
-            console.log('uniqueProblems',uniqueProblems)
-            recommendedData=Object.keys(uniqueProblems).map(key=>{
-              return {
-                ...uniqueProblems[key].data,
-                subHeading : `Complete this activity to use the ${uniqueProblems[key].join(', ')}`
-              }
-            })
+              });
+              recommendedData = Object.keys(uniqueActivities).map(key => {
+                return {
+                  ...uniqueActivities[key].data,
+                  subHeading:
+                    "Complete this activity to use the " +
+                    uniqueActivities[key].join(", ")
+                };
+              });
             }
 
-            return recommendedData.length>0 ? (
+            return recommendedData.length > 0 ? (
               <RecommendationListCard
+                dummyData={recommendedData}
                 key={recommendationKey}
-                dummyData={ recommendedData }
                 RecomType={
                   recommendationKey === "youtube" ? "youtube" : "python"
```

Comment:
@Brainenjii we might want to expand the recommendationKey to more than just `python` and `youtube`.